MBBSEmu/Program.cs

```diff
@@ -280,28 +294,29 @@ private void Run(string[] args)
                     return;
                 }
 
+                //If the user specified CLI mode, don't start the UI
+                if (!_cliMode)
+                {
+                    var mainMBBSEmuWindow = new MainView(_serviceResolver);
+
+                    //Start UI in a Task as to not block this thread
+                    Task.Run(() =>
+                    {
+                        mainMBBSEmuWindow.Setup();
+                        mainMBBSEmuWindow.Run();
+                    });
+
+                    //Wait for the UI to be running before continuing
+                    while (!mainMBBSEmuWindow.isRunning)
+                        Thread.Sleep(100);
```

Comment:
maybe sleep longer and add some sort of console out to let people know something is happening, like {
  Console.print("Waiting for main window to create");
  Thread.Sleep(1000);
}

Reply:
Made it sleep 500ms and it appends a `.` to the console message `Waiting for MBBSEmu GUI to start...`

I wouldn't expect this to append more than 1-2 period before the GUI thread fully starts